test/testMinsky.cc

```diff
@@ -189,6 +189,18 @@ SUITE(Minsky)
       CHECK_CLOSE(0.3, var["h"]->value(), 1e-4);
 
     }
+    
+//  TEST_FIXTURE(TestFixture,displayEquations)
```

Comment:
Not surprised this didn't work :)

Reply:
Hehe, it had me going around circles for a while this morning :)